File: API/API/ViewModel/TopicAnswers.cs
using API.Models;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace API.ViewModel
{
    public class TopicAnswers
    {
        public Topic Topic { get; set; }
        public List<Answer> Answers { get; set; }

        
    }
}

File: Android_Final/app/src/main/java/com/example/android_etpj/models/TopicAnswers.java
package com.example.android_etpj.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;
import java.util.List;

public class TopicAnswers implements Serializable {
    @SerializedName("Topic")
    private Topic topic;

    @SerializedName("Answers")
    public List<Answer> answers;

    public TopicAnswers() {
    }

    public Topic getTopic() {
        return topic;
    }

    public void setTopic(Topic topic) {
        this.topic = topic;
    }

    public List<Answer> getAnswers() {
        return answers;
    }

    public void setAnswers(List<Answer> answers) {
        this.answers = answers;
    }

    @Override
    public String toString() {
        return "TopicAnswers{" +
                "topic=" + topic +
                '}';
    }
}
